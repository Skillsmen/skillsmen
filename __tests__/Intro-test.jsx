/* eslint-env jest, amd */
// __tests__/Intro-test.js
import 'react-native';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
// import Intro from '../testComponents/Intro.jsx';
import Index from '../mobile/index';


it('renders correctly', () => {
  const tree = renderer.create(
    <Index />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
