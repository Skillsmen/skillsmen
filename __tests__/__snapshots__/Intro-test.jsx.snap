exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    Test test west
  </Text>
</View>
`;
